packages/resolver/src/adapters/unstoppable/api.ts

```diff
@@ -40,28 +40,23 @@ export const unstoppableApiGetCryptoAddress = (
 
         handleApiError(error)
       } else {
+        // parsing
         const safeParsedAdaResponse = UnstoppableApiAdaResponseSchema.safeParse(
           response.value.data,
         )
         const safeParsedGeneralResponse =
           UnstoppableApiGeneralResponseSchema.safeParse(response.value.data)
 
+        // checking
         const hasCardanoAddress = safeParsedAdaResponse.success
-        const hasOtherBlockchainAddress =
-          !hasCardanoAddress &&
-          safeParsedGeneralResponse.success &&
-          Object.keys(response.value.data.records).length > 0
-        const hasNotAnyAddress =
-          !hasOtherBlockchainAddress && safeParsedGeneralResponse.success
+        const hasOtherBlockchainAddress = safeParsedGeneralResponse.success
 
         if (hasCardanoAddress)
           return response.value.data.records['crypto.ADA.address']
 
         if (hasOtherBlockchainAddress)
           throw new Resolver.Errors.WrongBlockchain()
 
-        if (hasNotAnyAddress) throw new Resolver.Errors.NotFound()
-
         throw new Resolver.Errors.InvalidResponse()
```

Comment:
keeping this to cover the case when a wrong result is resolved

i.e.

`{ '...ADA...': undefined }`